extensions/wikia/DesignSystem/controllers/templates/DesignSystemGlobalFooter_imageHeader.php

```diff
@@ -1,9 +1,9 @@
-<h2 class="wds-global-footer__<?= $section ?>-header">
+<h2 class="wds-global-footer__<?= Sanitizer::encodeAttribute( $section ) ?>-header">
 	<?php if ( $model['type'] === 'link-image' ) : ?>
-		<a href="<?= $model['href'] ?>">
-			<?= DesignSystemHelper::getSvg( $model['image'], 'wds-global-footer__' . $section . '-logo' ) ?>
+		<a href="<?= Sanitizer::cleanUrl( $model['href'] ) ?>">
```

Comment:
`Sanitizer::cleanUrl` isn't intended for escaping URLs in a href attribute, it's intended for use by the parser, which then escapes the content again. It's not unsafe since it should URL encode the worst of it, but in this context, the best escaping method is still `Sanitizer::encodeAttribute`.